<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWindow</class>
 <widget class="QDialog" name="ResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>–†–µ–∑—É–ª—å—Ç–∞—Ç—ã</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
    font-size: 28px;
    font-weight: bold;
    padding: 20px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #3498db, stop:0.3 #2ecc71, stop:0.7 #f39c12, stop:1 #9b59b6);
    border-radius: 15px;
    border: 3px solid #2d4059;
}</string>
     </property>
     <property name="text">
      <string>üèÜ –†–µ–∑—É–ª—å—Ç–∞—Ç—ã –∏–≥—Ä—ã</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="winnerLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
   font-size: 16px;
    font-weight: bold;
    color: #e6e6e6;
    padding: 10px;
    background-color: #16213e;
    border-radius: 10px;
    border: 2px solid #3498db;
}</string>
     </property>
     <property name="text">
      <string>–ü–æ–±–µ–¥–∏—Ç–µ–ª—å:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="totalScoreLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #e6e6e6;
    padding: 15px;
    background-color: #16213e;
    border-radius: 10px;
    border: 2px solid #2ecc71;
}</string>
     </property>
     <property name="text">
      <string>–û–±—â–∞—è —Å—É–º–º–∞ –æ—á–∫–æ–≤:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #3498db, stop:1 #2980b9);
    color: white;
    font-size: 18px;
    font-weight: bold;
    border: none;
    border-radius: 10px;
    padding: 15px 30px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #2980b9, stop:1 #1c5a7d);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #1c5a7d, stop:1 #144a69);
}</string>
     </property>
     <property name="text">
      <string>üëç –ó–∞–∫—Ä—ã—Ç—å</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
